docs/notebooks/circular_regression.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>Is it possible to include a legend here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>